ivadomed/inference.py

```diff
@@ -110,10 +111,11 @@ def get_onehotencoder(context: dict, folder_model: str, options: dict, ds: Datas
     metadata_dict = joblib.load(Path(folder_model, 'metadata_dict.joblib'))
     for idx in ds.indexes:
         for i in range(len(idx)):
-            idx[i]['input_metadata'][0][context['FiLMedUnet']['metadata']] = options['metadata']
+            idx[i]['input_metadata'][0][context[ConfigKW.FILMED_UNET][ModelParamsKW.METADATA]] = options['metadata']
```

Comment:
So I think you missed the keyword conversion for the `options` dict. Which is generated initially at the `run_segment_command` [here](https://github.com/ivadomed/ivadomed/blob/0c2e6d7d0dd16feadd97929b342e1796a6c02b7d/ivadomed/main.py#L232)